pkg/network/openshift_sdn_test.go

```diff
@@ -296,3 +296,136 @@ func TestOpenShiftSDNMultitenant(t *testing.T) {
 		g.Expect(objs).To(ContainElement(HaveKubernetesID("NetNamespace", "", ns)))
 	}
 }
+
+func TestOpenshiftControllerConfig(t *testing.T) {
```

Comment:
this and `TestOpenshiftNodeConfig` seem like an unrelated patch?

Reply:
I just added it for parity with the node config, which I definitely wanted to test.